### PR TITLE
Mobile Experience Improvement

### LINKED ISSUE #5638: Mobile Experience Improvement
### Description
The Layer5 Community Handbook page at [Layer5 Community Handbook](https://layer5.io/community/handbook) needs improvements for better mobile experience. Currently, the content and buttons are not fit for smaller screen.

### Expected Behavior
Community handbook page to be fully responsive and user-friendly on mobile devices. Content should adjust properly, and buttons should be in size.

### Screenshots
Community

### Environment:
- Host OS:
- Browser:

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by Sakshampoply**
**Description**
Fixed the mobile content placement issues couldn't find any button issues

This PR fixes #5638 

**Notes for Reviewers**
Screenshot 2024-07-10 at 6 09 50 PM

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by hargunkaur286** [comment]
@Sakshampoply Please mention the issue number that you are trying to fix, not the PR number, infront of "This PR fixes #"

**Comment by Ashparshp** [comment]
@Sakshampoply Yes, please!

File: src/components/handbook-navigation/intra-page.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { Link } from "gatsby";
 import styled from "styled-components";
 
 const JoinCommunityWrapper = styled.div`
-  @media screen and (min-width: 751px) and (max-width: 1279px) {
+  @media screen and (min-width: 300px) and (max-width: 1279px) {
```
Comment (sudhanshutech) [inline src/components/handbook-navigation/intra-page.js:6]: is this by random you selected this resolution?

Comment (Sakshampoply) [reply src/components/handbook-navigation/intra-page.js:6]: I needed to make the mobile version responsive and most small mobiles are 300-350px in width

**Comment by vishalvivekm** [comment]
@Sakshampoply 
Thank you for your contribution.
Let's discuss this on Websites' call. Add this as an agenda item into the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit), if you would :)

**Review by Ashparshp — COMMENTED** [review]
@sudhanshutech, Have a suggestion? Or we can proceed further?

**Comment by leecalcote** [comment]
Thank you, @Sakshampoply!

**Review by sudhanshutech — APPROVED** [review]

**Pull request merged**